Static/Row.swift

```diff
@@ -134,7 +140,12 @@ public struct Row: Hashable, Equatable {
     // MARK: - Initializers
 
     public init(text: String? = nil, detailText: String? = nil, selection: Selection? = nil,
-        image: UIImage? = nil, accessory: Accessory = .None, cellClass: CellType.Type? = nil, context: Context? = nil, editActions: [EditAction] = [], UUID: String = NSUUID().UUIDString) {
+        image: UIImage? = nil, accessory: Accessory = .None, cellClass: CellType.Type? = nil, cellNib: UINib? = nil, height: CGFloat? = nil, context: Context? = nil, editActions: [EditAction] = [], UUID: String = NSUUID().UUIDString) {
+            
+        if (cellNib != nil && cellClass == nil)
```

Comment:
I have changed to the `if let` syntax that I also find better.
I have not found a way to access the nib name from the UINib object (UINib object .description : `<UINib: 0x7f8440505140>` :cry: ) so I have set the unwrapped UINib to `_`.  

On a side note, the only way I think of that can help us keep the name for debugging is to pass the UINib name String instead of the UINib object as parameter of the initializer, and instantiate the UINib object in it, but It feels kinda dirty to me as a new UINib is created each time and cannot be reused (In the case of a table view with x cells using the same nib for example)